docs/src/connecting/rocksdb.md

```diff
@@ -2,18 +2,12 @@
 
 RocksDb enables the preservation of the blockchain's state locally, facilitating its future utilization.
 
-To create a local database, follow these instructions:
+To create or to use a local database, follow these instructions:
 
 ```rust,ignore
-{{#include ../../../examples/contracts/src/lib.rs:create_rocksdb}}
-```
-
-To utilize an existing database, follow these instructions:
-
-```rust,ignore
-{{#include ../../../examples/cookbook/src/lib.rs:use_created_rocksdb}}
+{{#include ../../../examples/cookbook/src/lib.rs:create_or_use_rocksdb}}
 ```
 
 > Note 1: If the specified database does not exist a new database will be created at that path.
 
-> Note 2: To utilize the code snippets above, either the `fuel-core` binary must be present or the `rocksdb` feature needs to be used.
\ No newline at end of file
+> Note 2: To utilize the code snippets above, either the `fuel-core` binary must be present or the `fuel-core-lib, rocksdb` features need to be enabled.
```

Comment:
```suggestion
> Note 2: To utilize the code snippets above, either:
- the `fuel-core` binary must be present
or
- both the `fuel-core-lib` and `rocksdb` features need to be enabled.
```

Reply:
It does sound like you can use the `rocksdb` code without enabling the `rocksdb` feature if you have a `fuel-core` binary present. If that's true, I think we should prevent this, and make it only available if `rocksdb` is enabled.

Reply:
Why?We explained when to use `rocksdb` feature.

Reply:
What I mean is that the current phrasing is ambiguous, because it sounds like if I just have `the fuel-core binary present` then I can use `rocksdb` without enabling the `rocksdb` feature.

Reply:
Yes, it should behave that way. There is no need to add a flag if we don't need it.

Reply:
Oh ok I didn't understand that was expected behavior.